tests/python_functional/src/helpers/loggen/conftest.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python
+#############################################################################
+# Copyright (c) 2020 One Identity
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+import pytest
+from pathlib2 import Path
+from psutil import TimeoutExpired
+
+import src.testcase_parameters.testcase_parameters as tc_parameters
+from src.common.blocking import wait_until_true
+from src.executors.process_executor import ProcessExecutor
+from src.syslog_ng.syslog_ng_paths import SyslogNgPaths
+
+
+class Loggen(object):
+
+    def __init__(self, testcase_parameters):
+        self.loggen_proc = None
+        self.target = None
+        self.port = None
+        self.parameters = None
```

Comment:
I think `target`, `port` and `parameters` should be in the `start()` function scope, and they are not needed in `self`

Reply:
removed from the self structure